$MirOS: ports/net/bitlbee/patches/patch-bitlbee_h,v 1.2 2005/07/06 18:06:12 bsiegert Exp $
$OpenBSD: patch-bitlbee_h,v 1.2 2003/10/16 15:00:17 naddy Exp $
Use a more usefull name for the server
--- bitlbee.h.orig	Tue Dec 27 23:20:58 2005
+++ bitlbee.h	Tue Dec 27 23:20:22 2005
@@ -81,7 +81,7 @@
 
 #define _( x ) x
 
-#define ROOT_NICK "root"
+#define ROOT_NICK "bitlbee"
 #define ROOT_CHAN "&bitlbee"
 #define ROOT_FN "User manager"
 
